src/pages/ReimbursementAccount/RequestorOnfidoStep.js

```diff
@@ -31,6 +32,19 @@ class RequestorOnfidoStep extends React.Component {
     constructor(props) {
         super(props);
         this.submit = this.submit.bind(this);
+        this.goBack = this.goBack.bind(this);
+    }
+
+    goBack() {
+        const platform = getPlatform();
+        const isNative = platform === CONST.PLATFORM.IOS || platform === CONST.PLATFORM.ANDROID;
+        if (!isNative) {
+            const onfidoBack = document.querySelector('.onfido-sdk-ui-NavigationBar-back');
+            if (onfidoBack && !onfidoBack.classList.contains('onfido-sdk-ui-NavigationBar-disabled')) {
+                return onfidoBack.click();
+            }
+        }
+        this.props.onBackButtonPress();
```

Comment:
What you think about hoisting onfidoBackAction it to another file where we would have index.js and index.native.js so we won't have the wole platform checking? 

The `onBackButtonPress` can be passed as argument then, sth like:
```js
...
onBackButtonPress={() => onfidoBackAction(this.props.onBackButtonPress)}
...
```